### ISSUE TITLE
Default game parameters

### ISSUE THREAD

**Issue by ColonelGerdauf** [PrismLauncher/PrismLauncher#1022]
### Role

I have specific settings like FOV that I wish to maintain between instances including new ones.

### Suggestion

Have a `options.toml` file in the root of the userdata folder

### Benefit

General QOL; people will want certain default settings to never be enabled

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

### You may use the editor below to elaborate further.

A use case for this is the "auto-jump" feature that arrived since 1.11.1. Many will find that feature frustrating and do not ever want to see it enabled. However, this is not a universal feeling; some may appreciate the setting even on desktops, and even more prefer to leave it to manual settings in case anything changes.

This can be configured as a new tab in the Prism settings where people can manually enter data that they feel works for them, including experimental and legacy settings.

This kind of thing can also work with specialized setting such as those for Optifine: `options_optifine.toml` and the like for others that are known and use their own settings file.

**Comment by muzikbike** [comment]
Would this cover all Java Edition versions, since there are several options that previously existed but have sine been removed? (e.g. Show FPS, Twitch broadcasting settings from 1.7-1.8, anisotropic filtering, alternate blocks, snooper settings, 3D anaglyph, VBOs, chat preview, and probably some others)

If possible, modded options settings would also be useful for keybind that only exist in mods, so I don't need to keep rebinding the narrator to something else.

**Comment by ColonelGerdauf** [comment]
Yes, that is the intention: carry the set defaults to the first launch of any instance using any minecraft version. This list can include snapshot specific features, as well as removed features or those rendered redundant in an update at some point.

Currently the focus is on the main game. Allowing defaults for an arbitrary mod can be tricky due to config formatting (toml or json, or even something else?) as well as the need for one to write their own file in the defaults folder.
